pom.xml

```diff
@@ -522,7 +512,7 @@
             <configuration>
               <rules>
                 <requireJavaVersion>
-                  <version>[1.8,16)</version>
+                  <version>[1.8,22)</version>
```

Comment:
Just curious if you found this in a release note or somewhere. A comment on how to update this could help the next person updating maven but not required

Reply:
actually I will revert this part and do 21 support in a separate PR